lua/autorun/server/bounties.lua

```diff
@@ -27,7 +27,7 @@ end
 function AddBounty(ply,target,amount)
 	if !ply:IsPlayer() or !target:IsPlayer() or type(amount) != "number" then return end
 	amount = amount > 0 and amount or 0
-	if BountyLimit[ply:SteamID()] == nil then BountyLimit[ply:SteamID()] = 0 end
+	BountyLimit[ply:SteamID()] = BountyLimit[ply:SteamID()] or 0
```

Comment:
i mean you can remove this line (and the one in playerdeath) and replace all non-assignment references to BountyLimit[ply:SteamID()] with (BountyLimit[ply:SteamID()] or 0)